### ISSUE TITLE
Files missing after cancelling move due to EACESS error

### ISSUE THREAD

**Issue by UnknownWitcher** [coderaiser/cloudcmd#383]
- **Version** (`cloudcmd -v`): latest
- **Node Version** `node -v`:
- **OS** (`uname -a` on Linux): Linux pulu 5.15.0-48-generic 54-Ubuntu SMP Fri Aug 26 13:26:29 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
- **Browser name/version**: Firefox 105.0.1
- **Used Command Line Parameters**: default docker compose file with the addition off user=1000:1000
- **Changed Config**:

I forgot to change a folders ownership and moved some files to it, it prompted with the permissions error (believe it was EACCES), so I hit cancel. I then changed the folders ownership and went to move the files but they're now missing.

**Comment by coderaiser** [comment]
Looks like it’s related to [@cloudcmd/move-files](https://github.com/cloudcmd/move-files)

**Comment by UnknownWitcher** [comment]
**how many files you tried to copy?** I moved 76 files

**are they on the same partition?** they were and I was moving them to a folder on the same partition.

**could you reproduce this with [cloudcmd/move-files](https://github.com/cloudcmd/move-files)?** are you wanting me to install that and run the code?

Here's a longer explanation of exactly what happened.

Originally CC was configured as root, I made some folders and realised they had root permissions, so I configured the permissions by add `user: 1000:1000`, I forgot about the folder that had been created as root so when I tried to move files to that folder through CC on firefox it popped up telling me I didn't have access and when I cancelled the transfer the files were gone. I kinda expected the process to be cancelled intead of files being deleted...

**Comment by coderaiser** [comment]
Landed fix in [v16.6.0](https://github.com/coderaiser/cloudcmd/releases/tag/v16.6.0) 🎉.

Is it works for you?

**Comment by KTasliel** [comment]
I had the same issue, I'm using OpernMediaVault 7.4 on Debian 12 (BookWorm) and MergerFS. Also Cloud Commander is in Docker. 

While I'm trying to Rename Folder From Castlevania.bla.bla.bla... to Castlevania, I noticed it is trying to move files to somewhere else and I canceled. I notice 1 file was moved to root of docker (as /Castlevania) but other files are totally disappeared.

cloudcmd -v : Gives error.
node -v : v.22.11.0
uname -a : Linux fc0b95926a50 6.8.12-4-pve #1 SMP PREEMPT_DYNAMIC PMX 6.8.12-4 (2024-11-06T15:04Z) x86_64 GNU/Linux

from Help (F1) : Cloud Commander v18.2.1
